hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/PingHandler.java

```diff
@@ -39,6 +39,6 @@
     @Produces({ APPLICATION_JSON })
     @ApiOperation("A dummy operation returning the current date on the server.")
     public Response ping() {
-        return Response.ok(new StringValue(new Date().toString())).build();
+        return Response.ok(new Date().toString()).build();
```

Comment:
I guess you are right. btw. is anyone using it, isn't the `http://${host}/hawkular/inventory` with the list of available endpoints sufficient?